packages/pyxis-scss/src/scss/components/modal.scss

```
@@ -120,21 +116,15 @@ $_sizesOnBp: (
 }
 
 .modal__content {
-  @include foundations.horizontalPadding(m);
-  padding-bottom: foundations.spacing(xs);
-  padding-top: foundations.spacing(3xs);
-
-  @include foundations.mq(xsmall) {
-    padding: 0 foundations.spacing(2xl);
-  }
+  @include foundations.responsiveSpacing("padding", (3xs m xs), $fromXSmall: (0 xl));
 }
 
 .modal__footer {
-  @include foundations.padding(m);
+  @include foundations.responsiveSpacing("padding", m, $fromXSmall: (m 2xl));
```

Idem, controlla che sia giusto
```suggestion
  @include foundations.responsiveSpacing("padding", m, $fromXSmall: m 2xl, $fromMedium: m 3xl, $fromLarge: m 4xl);
```